.env

```diff
@@ -14,4 +14,4 @@ OPEN_SEA_API_KEY=
 BADGES_CONTRACT_ADDRESS=0x79215cdE52B4E8e4d46Dd81f45e38b88d2290815
 INFURA_PROJECT_ID=
 SENTRY_AUTH_TOKEN=
-DEFAULT_PIN=
+DEFAULT_PIN=123456
```

Comment:
WTF? :)

Reply:
I guess Gonzalo mentioned this before.
If I'm not mistaken, it's a quick way to login on DEV mode - it enter this pin for you.
@grilix, am I right?

Reply:
You are correct. It would be nice to have a way of adding custom environment variables on development without having to change this file, since it's a bit annoying having it as changed all the time 😞